src/sidebar/components/test/selection-tabs-test.js

```diff
@@ -87,28 +96,105 @@ describe('selectionTabs', function () {
     });
 
-    it('should display the new note button when the notes tab is active', function () {
+    it('should not display the new new note button when the annotations tab is active', function () {
       var elem = util.createDirective(document, 'selectionTabs', {
-        selectedTab: 'note',
+        selectedTab: 'annotation',
         totalAnnotations: '123',
         totalNotes: '456',
       });
       var newNoteElem = elem[0].querySelectorAll('new-note-btn');
-      assert.equal(newNoteElem.length, 1);
+
+      assert.equal(newNoteElem.length, 0);
     });
 
-    it('should not display the new new note button when the annotations tab is active', function () {
-      var elem = util.createDirective(document, 'selectionTabs', {
-        selectedTab: 'annotation',
-        totalAnnotations: '123',
-        totalNotes: '456',
+    context('when the sidebar tutorial is displayed', function () {
+      it('should display the shorter version of the no notes message when there are no notes', function () {
+        var elem = util.createDirective(document, 'selectionTabs', {
+          selectedTab: 'note',
+          totalAnnotations: '10',
+          totalNotes: 0,
+        });
+        var msg = elem[0].querySelectorAll('.annotation-unavailable-message__label')[0];
+
+        assert.include(msg.textContent, 'There are no page notes in this group.');
+        assert.notInclude(msg.textContent, 'Create one by clicking the');
+        assert.notInclude(msg.textContent, 'Create one by selecting some text and clicking the');
       });
 
-      var newNoteElem = elem[0].querySelectorAll('new-note-btn');
-      assert.equal(newNoteElem.length, 0);
+      it('should display the shorter version of the no annotations message when there are no annotations', function () {
+        var elem = util.createDirective(document, 'selectionTabs', {
+          selectedTab: 'annotation',
+          totalAnnotations: 0,
+          totalNotes: '10',
+        });
+        var msg = elem[0].querySelectorAll('.annotation-unavailable-message__label')[0];
+
+        assert.include(msg.textContent, 'There are no annotations in this group.');
+        assert.notInclude(msg.textContent, 'Create one by clicking the');
+        assert.notInclude(msg.textContent, 'Create one by selecting some text and clicking the');
+      });
+    });
+
+    context('when the sidebar tutorial is not displayed', function () {
+      fakeSession.state.preferences.show_sidebar_tutorial = false;
+      fakeSettings.enableExperimentalNewNoteButton = false;
+
+      it('should display the longer version of the no notes message when there are no notes', function () {
+        var elem = util.createDirective(document, 'selectionTabs', {
+          selectedTab: 'note',
+          totalAnnotations: '10',
+          totalNotes: 0,
+        });
+        var unavailableMsg = elem[0].querySelectorAll('.annotation-unavailable-message__label')[0];
```

Comment:
You can use `querySelector(selector)` rather than `querySelectorAll(selector)[0]` if you are only interested in the first element.